packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -177,18 +192,85 @@ export class AzureCommunicationCallWithChatAdapter implements CallWithChatAdapte
     };
 
     this.callAdapter.onStateChange(onCallStateChange);
+    /* @conditional-compile-remove(breakout-rooms) */
+    this.callAdapter.on('breakoutRoomsUpdated', async (eventData: BreakoutRoomsEventData) => {
+      if (!eventData.data) {
+        return;
+      }
+      if (eventData.type === 'join') {
+        await this.breakoutRoomJoined(eventData.data);
+      } else if (eventData.type === 'assignedBreakoutRooms') {
+        if (eventData.data.state === 'closed') {
+          this.returnFromBreakoutRoom();
```

Comment:
nit: await this (not actually relevant as nothing else executes after this line, but if someone adds to this line it could be easily missed so better to add it)

```suggestion
          await this.returnFromBreakoutRoom();
```